Dfe.Academies.Academisation.WebApi/Controllers/LegacyProjectController.cs

```diff
@@ -141,5 +144,32 @@ public async Task<ActionResult> AddNote(int id, AddNoteRequest note)
 				_ => throw new NotImplementedException()
 			};
 		}
+
+		/// <summary>
+		///     Deletes the provided note from the project with the specified ID
```

Comment:
Small thing - this summary is not what is happening. It looks like you are deleting the note that matches all criteria rather than id.

Reply:
"from the project with the specified ID" - the rest is DDD impl. (the value object doesn't have an ID)